src/wp-admin/includes/upgrade.php

```diff
@@ -874,6 +874,10 @@ function upgrade_all() {
 		upgrade_550();
 	}
 
+	if ( $wp_current_db_version < 49138 ) {
+		upgrade_560();
+	}
```

Comment:
@SergeyBiryukov I've always been curious - how do you define the version number specified in clauses like this? Similarly, how/when is `wp-includes/version.php` updated during development?